lib/autoupdate/agent/config.go

```diff
@@ -203,16 +204,16 @@ func validateConfigSpec(spec *UpdateSpec, override OverrideConfig) error {
 	if override.Group != "" {
 		spec.Group = override.Group
 	}
-	switch override.URLTemplate {
+	switch override.BaseURL {
 	case "":
 	case "default":
-		spec.URLTemplate = ""
+		spec.BaseURL = ""
 	default:
-		spec.URLTemplate = override.URLTemplate
+		spec.BaseURL = override.BaseURL
 	}
-	if spec.URLTemplate != "" &&
-		!strings.HasPrefix(strings.ToLower(spec.URLTemplate), "https://") {
-		return trace.Errorf("Teleport download URL must use TLS (https://)")
+	if spec.BaseURL != "" &&
+		!strings.HasPrefix(strings.ToLower(spec.BaseURL), "https://") {
+		return trace.Errorf("Teleport download URL must use TLS (https://): %q", spec.BaseURL)
```

Comment:
```suggestion
		return trace.Errorf("Teleport download base URL %s must use TLS (https://)", spec.BaseURL)
```
Quotes are escaped with structured logging and don't look great